### DISCUSSION TITLE
Welcome to RHVoice Discussions!

### DISCUSSION THREAD

**Discussion by alex19EP** [RHVoice/RHVoice#252] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with RHVoice.

**Comment by zvezdochiot** [comment]
:question: Is this (see above) a default prompt? Or this prompt of authors? Something it categorically does not suit and cannot.

**Reply by alex19EP** [reply]
> Something it categorically does not suit and cannot.

why?

**Comment by zvezdochiot** [comment]
@alex19EP say:
>why?

>>To get started, ... tell us about what you do with RHVoice.

**Reply by alex19EP** [reply]
so tell us, if you want of course.

in any case, discussions are now available and I'm very happy about that.

**Comment by abitrolly** [comment]
I've been told about RHVoice on a meetup about "Interfaces for Humans" that we've organized many many years ago. We've been looking for interesting people and found Alexander Severin, who teaches people who have lost sight how to access information. He revealed a story about Olga who are also sight impaired, and who had written an open source voice synthesizer in C++, because all other synthesizers were just too expensive for its users. I was so impressed with the presentation of this alternative world - how people browse the net, write music and complicated enterprise software, so I decided to help our Minsk RHVoice users to add Belarusian language. I didn't complete this goal yet. Through Alexander and community it is possible to record and label all sounds, but I could not really understand what exactly needs to be done and how it all works.

Nevertheless, it was an interesting trip into HMM and signal processing. I showed RHVoice to our Academy of Sciences and they started to look into HMM based synthesis. Then Sacrament company released HMM synthesis for Belarussian for Android. Our Linux community prepared patches for Python API and for Linux packaging. Very long PR that is I think is lost, because it is impossible to rebase. :D

While looking for info about Olga, I also discovered this podcast, which was very unusual because it talks about money among other things that are covered. People credits, sound editing, very touching compared to all content that I usually consume. And of course, Olga story. )

* Тифлочас. Выпуск 10. Синтезаторы речи: два подхода. Часть 1. Эфир от 20.03.2013 (from 11:20)
http://www.radiovos.ru/arch/Tiflo4as/Tiflochas_010_sintezatori_rechi.2_podhoda_ch.1.mp3
* Тифлочас. Выпуск 11. Синтезаторы речи: два подхода. Часть 2. Эфир от 27.03.2013
http://www.radiovos.ru/arch/Tiflo4as/Tiflochas_011_sintezatori_rechi.2_podhoda_ch.2.mp3

I am actually more interested in voice recognition, where HMM can also be used, although that needs custom training, and this part of pipeline in RHVoice was (is?) not sufficiently covered.
